apps/chat-e2e/src/testData/conversationHistory/conversationData.ts

```diff
@@ -493,12 +493,6 @@ export class ConversationData extends FolderData {
     return playbackConversation;
   }
 
-  public prepareDefaultSharedConversation() {
```

Comment:
Why? could you explain?

Reply:
this method is outdated and cannot be used to create a shared conversation